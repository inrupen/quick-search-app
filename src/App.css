@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');

.App {
    height: 100vh;
    font-family: 'Roboto', sans-serif;
}
