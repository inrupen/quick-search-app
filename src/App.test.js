import React from "react"
import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"
import App from "./App"

let AppWrapper

beforeEach(() => {
    AppWrapper = render(<App />)
})

afterEach(() => {
    AppWrapper = render(<App />)
})

describe("root app", () => {
    it("App renders successfully", () => {
        expect(AppWrapper).toMatchSnapshot();
    })
})
