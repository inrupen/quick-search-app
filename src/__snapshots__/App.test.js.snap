// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root app App renders successfully 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="sc-eCstlR ewZxTX"
        >
          <div
            class="sc-bdfBQB kvNLcG"
            data-testid="header"
          >
            Search app
          </div>
          <div
            class="search"
          >
            <div
              class="sc-dlfnuX kWdcCQ"
            >
              <input
                aria-label="Search through sites content"
                class="sc-hKgJUU cEVLHO"
                data-testid="search"
                id="search"
                maxlength="20"
                placeholder="Zoeken"
                type="search"
                value=""
              />
              
              
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="sc-eCstlR ewZxTX"
      >
        <div
          class="sc-bdfBQB kvNLcG"
          data-testid="header"
        >
          Search app
        </div>
        <div
          class="search"
        >
          <div
            class="sc-dlfnuX kWdcCQ"
          >
            <input
              aria-label="Search through sites content"
              class="sc-hKgJUU cEVLHO"
              data-testid="search"
              id="search"
              maxlength="20"
              placeholder="Zoeken"
              type="search"
              value=""
            />
            
            
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
